src/CMakeLists.txt

```
@@ -138,3 +138,8 @@ if (NFD_INSTALL)
     FILE ${TARGET_NAME}-config.cmake
   )
 endif()
+
+option(NFD_OVERRIDE_RECENT_WITH_DEFAULT "Use defaultPath instead of recent folder on Windows" ON)
+if (NFD_OVERRIDE_RECENT_WITH_DEFAULT)
+    target_compile_definitions(${TARGET_NAME} PRIVATE nfd_OVERRIDE_RECENT_WITH_DEFAULT)
```

```suggestion
    target_compile_definitions(${TARGET_NAME} PRIVATE NFD_OVERRIDE_RECENT_WITH_DEFAULT)
```

nit: Other compiler defines seem to use all caps, so let's follow that convention.